src/components/views/Sheets/Sheets.tsx

```diff
@@ -230,9 +225,8 @@ const Content = styled.div`
 `
 
 const SmActions = styled.nav`
-  position: fixed;
-  inset-inline: ${({ theme }) => 2 * theme.spacingBase}rem;
-  inset-block-end: ${({ theme }) => 3 * theme.spacingBase}rem;
+  margin-inline: ${({ theme }) => 2 * theme.spacingBase}rem;
+  margin-block: ${({ theme }) => theme.spacingBase}rem ${({ theme }) => 2 * theme.spacingBase}rem;
```

Comment:
I would do it like this here which I think is a bit easier to read, but you can choose!

```suggestion
  margin-block-start: ${({ theme }) => theme.spacingBase}rem;
  margin-block-end: ${({ theme }) => 2 * theme.spacingBase}rem;
```